docs/configure.md

```diff
@@ -393,67 +393,66 @@ See [the official documentation](https://kubernetes.io/docs/concepts/configurati
 
 ## Configure a storage class
 
-Sourcegraph expects there to be storage class named `sourcegraph` that it uses for all its persistent volume claims. This storage class must be configured before applying the base configuration to your cluster. The configuration details differ depending on your hosting provider, so you should:
+Sourcegraph expects there to be storage class named `sourcegraph` that it uses for all its persistent volume claims. This storage class must be configured before applying the base configuration to your cluster.
 
-1. Create a stub `base/sourcegraph.StorageClass.yaml`.
+Create `base/sourcegraph.StorageClass.yaml` with the appropriate configuration for your cloud provider and commit the file to your fork.
 
-   ```yaml
-   # base/sourcegraph.StorageClass.yaml
-   kind: StorageClass
```

Comment:
If you are going to delete this snippet, you should explicitly document that you need the `deploy: sourcegraph` label.

Reply:
Moved to "Other cloud providers" section.